components/admin/formSteps/BannerDetailsForm.tsx

```
@@ -13,10 +14,54 @@ type BannerDetailsFormProps = {
+      const res = await axios.post(
+        "https://api.starknet.quest/admin/upload_image", 
+        formData,
+        {
+          headers: { "Content-Type": "multipart/form-data" },
+        }
+      );
+      const imageUrl = res.data.url;
+      setQuestInput((prev) => ({
+        ...prev,
+        banner: {
+          tag: (prev.banner ?? {}).tag ?? "",
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)